cmd/portforward.go

```diff
@@ -0,0 +1,384 @@
+// Copyright 2021 Northern.tech AS
+//
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+//
+//        http://www.apache.org/licenses/LICENSE-2.0
+//
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/signal"
+	"strconv"
+	"strings"
+	"time"
+
+	"github.com/mendersoftware/go-lib-micro/ws"
+	wspf "github.com/mendersoftware/go-lib-micro/ws/portforward"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+	"github.com/vmihailenco/msgpack"
+	"golang.org/x/sys/unix"
+
+	"github.com/mendersoftware/mender-cli/client/deviceconnect"
+)
+
+const (
+	argBindHost    = "bind"
+	readBuffLength = 4096
+	localhost      = "127.0.0.1"
+)
+
+var portForwardCmd = &cobra.Command{
+	Use:   "port-forward [DEVICE_ID] [LOCAL_PORT:REMOTE_PORT] ...",
```

Comment:
`DEVICE_ID` is not optional, to be consistent with the rest of the tool it should be:
```suggestion
	Use:   "port-forward DEVICE_ID [LOCAL_PORT:REMOTE_PORT] ...",
```

or even better something like:
```suggestion
	Use:   "port-forward DEVICE_ID [tcp|udp/]LOCAL_PORT[:REMOTE_PORT] [[tcp|udp/]LOCAL_PORT[:REMOTE_PORT]...]",
```